docker-compose.yml

```diff
@@ -39,31 +40,57 @@ services:
       - UWSGI_OFFLOAD_THREADS=1
       - UWSGI_HARAKIRI=15
       - UWSGI_DIE_ON_TERM=1
-      - UWSGI_MODULE=settings.wsgi:application
+      - UWSGI_MODULE=main.wsgi:application
+      - UWSGI_PY_AUTORELOAD=1
+
+  test:
+    build:
+      context: .
+      target: tests
+      args:
+        - SECRET_KEY=dev
+        - AUTHORIZATION_TOKEN=auth-token
+        - GET_AUTHORIZATION_TOKEN=get-auth-token
+    volumes:
+      - ./src:/src
+      - ./tests:/tests
+      - ./deploy:/deploy
+    environment:
+      - SECRET_KEY=dev
+      - AUTHORIZATION_TOKEN=auth-token
+      - GET_AUTHORIZATION_TOKEN=get-auth-token
+      - DATABASE_NAME=waarnemingen_mensen
+      - DATABASE_HOST=database
+      - DATABASE_USER=waarnemingen_mensen
+      - DATABASE_PASSWORD=insecure
```

Comment:
Niet nodig als je een goede default zet in je settings die overeenkomt met de default in dockercompose. (Andere projecten gebruiken "dev" voor alle waardes).